adcircpy/server/slurm_config.py

```diff
@@ -18,6 +18,7 @@ def __init__(
             filename: str = 'slurm.job',
             run_directory: str = '.',
             run_name: str = None,
+            slurm_nodes: int = None,
```

Comment:
This was already implemented as "nodes" argument.

Reply:
where is that?

Reply:
see line 28 on previous

Reply:
looks like we both made the same commit on different branches, funny
I've removed `slurm_nodes`

Reply:
I did mention I had fixed this...